File: src/main/java/com/wanderers/hotelier_webservices/server/service/api/HotelierService.java
package com.wanderers.hotelier_webservices.server.service.api;

import com.wanderers.hotelier_webservices.server.exception.HotelierServiceException;

public interface HotelierService {

    /**
     * Check if the hotelier exist in the datasource
     *
     * @param hotelierId
     * @return
     * @throws HotelierServiceException
     */
    Boolean isExistingHotelier(String hotelierId) throws HotelierServiceException;
}

File: src/main/kotlin/com/wanderers/hotelier_webservices/server/service/api/HotelierService.kt
package com.wanderers.hotelier_webservices.server.service.api

import com.wanderers.hotelier_webservices.server.exception.HotelierServiceException

interface HotelierService {
    /**
     * Check if the hotelier exist in the datasource
     *
     * @param hotelierId
     * @return
     * @throws HotelierServiceException
     */
    @kotlin.Throws(HotelierServiceException::class)
    fun isExistingHotelier(hotelierId: String?): Boolean?
}